pkg/vpcmodel/grouping.go

```diff
@@ -165,6 +175,39 @@ func (g *groupedConnLine) getSrcOrDst(isSrc bool) EndpointElem {
 	return g.dst
 }
 
+// isOverApproximated() checks if the line was over approximated during the load balancer abstraction
+// it uses the lb AbstractionInfo that was kept during the approximation
```

Comment:
over-approximated means there are missing connection?
please use the same terminology
```suggestion
// isOverApproximated() checks if the line was over approximated - namely, has missing connection, during the load balancer abstraction
// it uses the lb AbstractionInfo that was kept during the approximation
```

Reply:
fixed